plugins/importexport/doaj/DOAJPlugin.inc.php

```diff
@@ -103,6 +144,314 @@ function exportJournal(&$journal, $outputFile = null) {
 		}
 		return true;
 	}
-}
 
-?>
+	/**
+	 * @param $request PKPRequest
+	 * @param $selectedObjects array
+	 */
+	function _markRegistered($request, $selectedObjects) {
+
+		$target = $request->getUserVar('target');
+		$action = $target . 's';
```

Comment:
Generally speaking I'm not fond of programmatically building function names like this -- this case has aleady caused a mess in the CrossRef plugin when "all" was added as an option, then pluralized as "alls". I'd suggest using switch/case or whitelist or something clearer. It's also hard to grep for uses of functions when they're built this way.